docs/scarpet/api/Auxiliary.md

```diff
@@ -4,33 +4,48 @@ Collection of other methods that control smaller, yet still important aspects of
 
 ## Sounds
 
+Functions which allow to manipulate the player's auditory environment.
+
 ### `sound()`, `sound(name, pos, volume?, pitch?, mixer?)`
 
 Plays a specific sound `name`, at block or position `pos`, with optional `volume` and modified `pitch`, and under
 optional `mixer`. Default values for `volume`, `pitch` and `mixer` are `1.0`, `1.0`, and `master`. 
 Valid mixer options are `master`, `music`, `record`, `weather`, `block`, `hostile`,`neutral`, `player`, `ambient`
 and `voice`. `pos` can be either a block, triple of coords, or a list of thee numbers. Uses the same options as a
- corresponding `playsound` command.
+ corresponding `playsound` command. Note that modifying `pitch` also modifies the duration of the sound.
  
-Used with no arguments, return the list of available sound names.
+Used with no arguments, return the list of all available sound names.
  
 Throws `unknown_sound` if sound doesn't exist.
+<pre>
+//Plays the sound of a zombie breaking down a door, with 4x the volume and 1/4th the pitch
+//Turning down ambient music in sound settings will reduce the volume of this sound
+sound('entity.zombie.break_wooden_door', [x, y, z], 4, 0.25, 'ambient')
+</pre>
 
 ## Particles
 
+Functions which allow to send particles of various forms to the player. These allow to highlight zones for all players,
+including those with vanilla clients (unlike with `draw_shape` function which only works for players with carpet clients)
```

Comment:
Not 100% true, many shapes have fallbacks, there's just a few that don't (most if not all _could_ have a fallback, just it's too complicated/no one bothered). IMO this comment should be in `draw_shape` if it's not there.

Reply:
ye true